src/js/components/TextInput/TextInput.js

```diff
@@ -232,6 +233,22 @@ const TextInput = forwardRef(
       if (onSuggestionsClose) onSuggestionsClose();
     }, [messages.onSuggestionsClose, onSuggestionsClose]);
 
+    const setFromSuggestion = (event, suggestion) => {
```

Comment:
This function name felt a little unclear to me. Maybe `setValueFromSuggestion` ?

Reply:
good idea, changed